packages/@webex/plugin-cc/src/services/HttpRequest.ts

```diff
@@ -96,14 +96,24 @@ class HttpRequest {
     try {
       const {service, resource, method, payload, eventType, success, failure} = options;
 
+      // prune undefined values
+      Object.keys(payload).forEach((key) => payload[key] === undefined && delete payload[key]);
+
       // Send the service request
-      const response = await this.webex.request({
+      const response: IHttpResponse = await this.webex.request({
```

Comment:
Do we need to specify the type here? Based on the value assigned, type will be inferred automatically